sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientMockTests.cs

```diff
@@ -310,7 +310,7 @@ public async Task StartRecognizeBusinessCardsSendsUserSpecifiedLocale(string loc
             var options = new FormRecognizerClientOptions() { Transport = mockTransport };
             var client = CreateInstrumentedClient(options);
 
-            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.InvoiceLeTiff);
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.BusinessCardJpg);
             var recognizeOptions = new RecognizeReceiptsOptions { Locale = locale };
             await client.StartRecognizeReceiptsAsync(stream, recognizeOptions);
```

Comment:
It seems we're calling the wrong method here (receipts instead of business cards).

Reply:
another great catch